### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Emmarie-Ahtunan**
adding correct alt-text for screen readers.

### What change does this PR introduce?

Adding correct alt-text for optimized screen readers.

### Why was this change needed?

Increased accessibility on README.md

### Other information (Screenshots)

You might want to consider adding [GitHub's accessibility alt-text-bot](https://github.com/marketplace/actions/accessibility-alt-text-bot) for improved automated accessibility.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <a href="https://novu.co" target="_blank">
   <picture>
     <source media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/2233092/213641039-220ac15f-f367-4d13-9eaf-56e79433b8c1.png">
-    <img src="https://user-images.githubusercontent.com/2233092/213641043-3bbb3f21-3c53-4e67-afe5-755aeb222159.png" width="280" alt="Logo"/>
+    ![Novu Logo]<img src="https://user-images.githubusercontent.com/2233092/213641043-3bbb3f21-3c53-4e67-afe5-755aeb222159.png" width="280"/>
```
Comment (p-fernandez) [inline README.md:5]: Aren't we missing here also the `alt-text`?

**Review by p-fernandez — COMMENTED** [review]
Thanks for your contribution. 🌟 

Very important point regarding accessibility. It is an area we need to improve.

File: README.md (hunk 1 above)
Comment (Emmarie-Ahtunan) [reply README.md:5]: the ![] is the standard for alt-text in markdown. For some reason, I wasn't able to use the standard ![] for the rest of the alt-text in the README.md.

Comment (p-fernandez) [reply README.md:5]: No worries. Wanted to double check as a11y is not my strongest field.

**Review by p-fernandez — APPROVED** [review]
Many thanks 🌟

File: README.md (hunk 1 above)
Comment (Emmarie-Ahtunan) [reply README.md:5]: if you ever want or need a11y help, just give me a shout. it's my specialty.

**Pull request merged**